### ISSUE TITLE
What set are accuracy results on?

### ISSUE THREAD

**Issue by alecda573** [lck1201/simple-effective-3Dpose-baseline#7]
General question the 44 mm error you give is that on the test set of on the training set?

**Comment by lck1201** [comment]
test set
